Jenkinsfile

```diff
@@ -342,6 +368,13 @@ pipeline {
 
   post {
     failure {
+      script {
+        if (env.BRANCH_NAME.startsWith('PR-')) {
```

Comment:
Is there a reason for posting GitHub comment only if the branch name start with  PR-